### PR TITLE
Remove duplicate networks

### PR DESCRIPTION
**Pull request by gagansh7171**
networks are repeated at lines 557 and 504

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -553,10 +553,6 @@ services:
     - intranet-server
     - internet-server
 
-    # Connect to the custom default network
-    networks:
-    - network
-
```
Comment (dhruvkb) [inline docker-compose.yml:559]: Please remove the earlier occurrence and keep this one to preserve the order that the `networks` key appears after the `depends_on` key.

**Review by dhruvkb — CHANGES_REQUESTED** [review]

**Comment by dhruvkb** [comment]
Oh, and good catch! 👍

File: docker-compose.yml (hunk 1 above)
Comment (gagansh7171) [reply docker-compose.yml]: done 👍

File: docker-compose.yml [hunk 2]
```diff
@@ -556,7 +552,7 @@ services:
     # Connect to the custom default network
     networks:
     - network
-
+
```
Comment (dhruvkb) [inline docker-compose.yml:555]: Please don't add/leave trailing whitespaces. Use an editor that [highlights them](https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces).

**Review by dhruvkb — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (gagansh7171) [reply docker-compose.yml]: done, thanks for the tip. I was pretty careless with that one.

**Review by algomaster99 — APPROVED** [review]

**Comment by gagansh7171** [comment]
@algomaster99 this pr is kinda important for the cli to work.

**Review by algomaster99 — APPROVED** [review]

**Pull request merged**